.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# All markdown files
+*.md @xmarcalx @kalyazin @pb8o @sudanl0
+
+# But not the ones in docs/
+docs/*.md
+
+# Except these specific ones
+docs/getting-started.md @xmarcalx @kalyazin @pb8o @sudanl0
+docs/prod-host-setup.md @xmarcalx @kalyazin @pb8o @sudanl0
+
+# Also cover all "*-policy.md" documents
+**/*policy.md @xmarcalx @kalyazin @pb8o @sudanl0
+**/*POLICY.md @xmarcalx @kalyazin @pb8o @sudanl0
+
+# Also these non-md files in the repository root
+THIRD_PARTY @xmarcalx @kalyazin @pb8o @sudanl0
+LICENSE @xmarcalx @kalyazin @pb8o @sudanl0
```

Comment:
Should we also add PGP-KEY.asc NOTICE ?

Reply:
Sounds reasonable. On an unrelated note, our NOTICE has "Copyright 2017 - 2020", which seems... wrong?